src/app/app.hpp

```diff
@@ -84,21 +85,54 @@ class App {
   MetricsData m_metrics_data;
 
   struct Empty {
+    explicit Empty(Context context, const std::optional<BGRAFrame>& background) 
+      : m_context(context)
+    {
+      // do deepcopy of BGRAframe 
+      if (background.has_value()) {
+        auto height = background->size.height();
+        auto width = background->size.width();
+        auto n = height * width * 4;
+        m_background = BGRAFrame{};
+        m_background->data = std::make_unique<u8[]>(n);
+        memcpy(m_background->data.get(), background->data.get(), n);
+        m_background->size = Size{ height, width };
+      }
+      else {
+        m_background = std::nullopt;
```

Comment:
It is initialized as `nullopt` by default

Reply:
fixed